src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -85,6 +88,18 @@ internal partial class AzureDevOpsIntegration : IDevOpsIntegration
         private const string BASE_VSO_URL = "https://app.vssps.visualstudio.com";
         #endregion
 
+        internal class DialogHost : IDialogHost
```

Comment:
not used, was playing with defining our own DialogHost - we can pass this in to showDialog